packages/augur-ui/src/modules/common/constants.ts

```diff
@@ -232,6 +232,19 @@ export enum MARKET_SORT_PARAMS {
   LAST_LIQUIDITY_DEPLETED = 'lastLiquidityDepleted',
 }
 
+export const SORT_OPTIONS = [
+  { value: MARKET_SORT_PARAMS.LIQUIDITY, header: 'Highest liquidity' },
+  { value: MARKET_SORT_PARAMS.OPEN_INTEREST, header: 'Highest open interest' },
+  { value: MARKET_SORT_PARAMS.VOLUME, header: 'Highest volume' },
+  { value: MARKET_SORT_PARAMS.CREATION_TIME, header: 'Recently created' },
+  { value: MARKET_SORT_PARAMS.END_DATE, header: 'Ending soon ' },
+  { value: MARKET_SORT_PARAMS.RECENTLY_TRADED, header: 'Recently Traded' },
+  {
```

Comment:
this one can be removed. There's a clean up ticket to do so. but might as well remove here.